buildpack.toml

```diff
@@ -0,0 +1,9 @@
+api = "0.1"
```

Comment:
i know you mentioned sticking with 0.1, but since you're not using the build plan, it should be easy to upgrade to 0.2

Reply:
Yeah, I think it has to be `0.1` for now, since we don't have a `0.2` compatible builder just yet.

Reply:
oh yea, duh 👍